### ISSUE TITLE
Download & enable rulesets during installation of seekrets

### ISSUE THREAD

**Issue by LinuxBozo** [18F/laptop#75]
As a new user to `git-seekret` I want to be able to run one script to install everything I need to start using these hooks to stop committing sensitive information.

Implementation:
See TODO items in [`seekrets-install`](https://github.com/18F/laptop/blob/seekret/seekrets-install). Initially, the rules can be downloaded using a reference to the `seekret` branch of this repo for testing, will need to reference `master` for merge. 
Added bonus points related to enabling rules: apuigsech/git-seekret#12

If fixing, please use `seekret` branch so it's merged into PR #69

**Comment by alain-hoang** [comment]
Started a clean slate re-install.  Upon further investigation a dependency has been moved to a different location.   Submitted a [pull request](https://github.com/apuigsech/seekret-source-git/pull/3) to upstream.

**Comment by alain-hoang** [comment]
Upstream has merged the [PR fix](https://github.com/apuigsech/seekret-source-git/pull/3) fix that addresses the missing dependency.   However attempts to try to rebuild project still resulting in errors:

```
# github.com/apuigsech/git-seekret/vendor/github.com/apuigsech/seekret-source-git
vendor/github.com/apuigsech/seekret-source-git/load_source.go:5: imported and not used: "github.com/apuigsech/git-seekret/vendor/github.com/emptyinterface/sshconfig"
vendor/github.com/apuigsech/seekret-source-git/load_source.go:127: too many arguments in call to models.NewObject
vendor/github.com/apuigsech/seekret-source-git/load_source.go:142: too many arguments in call to models.NewObject
vendor/github.com/apuigsech/seekret-source-git/load_source.go:192: too many arguments in call to models.NewObject
vendor/github.com/apuigsech/seekret-source-git/load_source.go:212: undefined: ssh_config in ssh_config.Parse
```

What appears to be happening is that the installed dependency for `seekret` is pointing to an old referenced version of seekret that is not using the updated object model.

cc @jcscottiii

**Comment by LinuxBozo** [comment]
@alain-hoang you can create a PR to update the upstream library using `glide update`.

**Comment by alain-hoang** [comment]
Running `glide update` does not update the upstream library that requires being updated.   I can manually edit the glide generated files but the current workaround to build `git-seekret` is to just use plain `go get` commands.

**Comment by LinuxBozo** [comment]
@alain-hoang should be good to go once https://github.com/apuigsech/git-seekret/pull/19 is merged.

**Comment by alain-hoang** [comment]
Thank you @LinuxBozo!   I had a similar [changeset](https://github.com/alain-hoang/git-seekret/commit/48e55cd0b30ecbc72a4f44463adacf57d5ebfb00) that I created.  Your version is more explicit in for all dependencies.  I think this will be more manageable once upstream has tagged releases to reference.

**Comment by alain-hoang** [comment]
Upstream has merged https://github.com/apuigsech/git-seekret/pull/19
